packages/drivers/web-utils/src/config.ts

```diff
@@ -0,0 +1,154 @@
+import { SelectorCriteria, Rect } from "./types";
+
+type CriteriaConfig<T> = {
+  extract: (el: HTMLElement) => T | null;
+  compare: (
```

Comment:
let's add an assertion on the fact that compare must return a normalized value between 0 and 1 (in the find-element method)